### PR TITLE
Updated remaining Project Configurator references to Project Manager and consolidated JsonUtils API usage.

### PR DESCRIPTION
**Pull request by cgalvan**
Updated all remaining references to Project Configurator with its replacement, Project Manager.

UPDATE: During PR feedback noticed that several of the APIs in AzFramework::FileFunc had been duplicated in AZ::JsonSerializationUtils, so replaced those usages.

Signed-off-by: Chris Galvan <chgalvan@amazon.com>

### CODE REVIEW

**Review by mbalfour-amzn — APPROVED** [review]

File: Code/Framework/AzFramework/AzFramework/FileFunc/FileFunc.cpp [hunk 1]
```diff
@@ -126,9 +126,9 @@ namespace AzFramework
             *      1. Key exists, value is identical -> do nothing
             *          This can occur if a user modifies a config file while the tooling is open.
             *          Example:
-            *              1) Load Project Configurator.
+            *              1) Load Project Manager.
             *              2) Modify your bootstrap.cfg to a different project.
-            *              3) Tell Project Configurator to set your project to the project you set in #2.
+            *              3) Tell Project Manager to set your project to the project you set in #2.
```
Comment (AMZN-nggieber) [inline Code/Framework/AzFramework/AzFramework/FileFunc/FileFunc.cpp:131]: Project Manager doesn't edit bootstrap.cfg or other cfg files

File: Code/Tools/SceneAPI/SceneCore/Components/SceneSystemComponent.h [hunk 2]
```diff
@@ -19,7 +19,7 @@ namespace AZ
         {
             // Scene system components are components that can be used to create system components
             //      in situations where the full initialization and/or construction of regular
-            //      system components don't apply such as in the Project Configurator's advanced
+            //      system components don't apply such as in the Project Manager's advanced
```
Comment (AMZN-nggieber) [inline Code/Tools/SceneAPI/SceneCore/Components/SceneSystemComponent.h:22]: There is no parity for this in Project Manager

File: Gems/SceneProcessing/Code/Source/Config/Components/SceneProcessingConfigSystemComponent.cpp [hunk 3]
```diff
@@ -34,7 +34,7 @@ namespace AZ
             ActivateSceneModule(SceneProcessing::s_sceneDataModule);
             ActivateSceneModule(SceneProcessing::s_sceneBuilderModule);
             
-            // Defaults in case there's no config setup in the Project Configurator.
+            // Defaults in case there's no config setup in the Project Manager.
```
Comment (AMZN-nggieber) [inline Gems/SceneProcessing/Code/Source/Config/Components/SceneProcessingConfigSystemComponent.cpp:37]: I don't believe Project Manager sets this up but I'm not sure what config it is referring to

File: Gems/SceneLoggingExample/Code/SceneLoggingExampleModule.cpp [hunk 4]
```diff
@@ -42,7 +42,7 @@ namespace SceneLoggingExample
         }
 
         // In this example, no system components are added. You can use system components 
-        // to set global settings for this gem from the Project Configurator.
+        // to set global settings for this gem from the Project Manager.
```
Comment (AMZN-nggieber) [inline Gems/SceneLoggingExample/Code/SceneLoggingExampleModule.cpp:45]: Unless this means enabling or disabling a gem Project Manager can't do that

**Review by AMZN-nggieber — CHANGES_REQUESTED** [review]
So of the usages are no longer applicable to Project Manager otherwise all good.

File: Code/Framework/AzFramework/AzFramework/FileFunc/FileFunc.cpp (hunk 1 above)
Comment (cgalvan) [reply Code/Framework/AzFramework/AzFramework/FileFunc/FileFunc.cpp:131]: This method actually is no longer used, so will just delete it

File: Code/Tools/SceneAPI/SceneCore/Components/SceneSystemComponent.h (hunk 2 above)
Comment (cgalvan) [reply Code/Tools/SceneAPI/SceneCore/Components/SceneSystemComponent.h:22]: Thanks will remove this mention

File: Gems/SceneProcessing/Code/Source/Config/Components/SceneProcessingConfigSystemComponent.cpp (hunk 3 above)
Comment (cgalvan) [reply Gems/SceneProcessing/Code/Source/Config/Components/SceneProcessingConfigSystemComponent.cpp:37]: Will remove this mention

File: Gems/SceneLoggingExample/Code/SceneLoggingExampleModule.cpp (hunk 4 above)
Comment (cgalvan) [reply Gems/SceneLoggingExample/Code/SceneLoggingExampleModule.cpp:45]: Will remove this mention outright as well

**Review by tjmichaels — APPROVED** [review]

**Review by AMZN-nggieber — APPROVED** [review]

**Pull request merged**